profiler/src/main/java/com/splunk/opentelemetry/profiler/StackTraceFilter.java

```diff
@@ -34,9 +34,15 @@
         "\"Common-Cleaner\""
       };
   private final boolean includeAgentInternals;
+  private final boolean includeJVMInternals;
```

Comment:
Nit: I think we've been strictly following camelCase everywhere, abbreviations included:
```suggestion
  private final boolean includeJvmInternals;
```

Reply:
Ok, cool.